groups/committee-steering/groups.yaml

```diff
@@ -43,12 +46,12 @@ groups:
     owners:
       - augustus@cisco.com
       - bentheelder@google.com
-      - cblecker@gmail.com
-      - ctadeu@gmail.com
       - k8s@auggie.dev
       - killen.bob@gmail.com
+      - maszulik@redhat.com
       - pal.nabarun95@gmail.com
-      - tpepper@vmware.com
+      - patrick.ohly@intel.com
```

Comment:
@pohly -- Please confirm preferred email address for Steering comms

Reply:
/assign @pohly

Reply:
Yes, that's the right one.